src/helix-ui/elements/_hx-alert.less

```diff
@@ -0,0 +1,70 @@
+@import "./_hx-element";
+@import "core/hx-icon";
+
+#close,
+#error,
+#info,
+#success,
+#warning {
+  padding-bottom: 0.875em;
+  padding-top: 0.875em;
+}
+
+#error,
+#info,
+#success,
+#warning {
+  flex-basis: 1em;
+  flex-grow: 0;
+  flex-shrink: 0;
+  margin-left: 1em;
+}
+
+#error,
+#success,
+#warning {
+  display: none;
+}
+
+#close {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  display: none;
+  margin-right: 1rem;
```

Comment:
Any reason for using `rem` here when the other margins are all `em`?

Reply:
I've changed all the references to `rem` for consistency, as I believe that is the unit of measurement we should be using.